### DISCUSSION TITLE
Is it possible to use Openlayers in Node environment?

### DISCUSSION THREAD

**Discussion by mkoskl** [openlayers/openlayers#16158] [Q&A]
Hello,

is it possible to use Openlayers in Node?

What I would like to do is to render the map and save the screenshot of the rendered map programmatically, i.e **not** using browser.

Br, Miika

**Comment by AfterFX** [comment]
Hello, Have you tried using `Puppeteer`? 
`Puppeteer uses to open the HTML file and capture a screenshot.`

**Reply by mkoskl** [reply]
Hi, @AfterFX, thanks, but that's also something I try to avoid. I was wondering whether it whould be possible to use Openlayers using just plain Node. Br, Miika

**Reply by AfterFX** [reply]
I doubt that you will manage to do something like that without DOM. Good luck.
